app/com/linkedin/drelephant/spark/fetchers/SparkFetcher.scala

```diff
@@ -67,6 +67,8 @@ class SparkFetcher(fetcherConfigurationData: FetcherConfigurationData)
     } else if (useRestForLogs) EventLogSource.Rest else EventLogSource.WebHdfs
   }
 
+  private lazy val shouldProcessLogsLocally = Option(fetcherConfigurationData.getParamMap.get("should_process_logs_locally")).exists(_.toBoolean)
```

Comment:
This will throw an exception if _.toBoolean fails.  Perhaps use exists(_.toLowercase == "true") instead?